app/views/controllers/projects/aliases/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<%
+  add_project_banner(@project)
+  @container = :wide
+  project_id = @project.id
+%>
+
+<h1>Project Aliases</h1>
+
+<table class="table table-striped table-project-members mt-3">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Target Type</th>
+      <th>Target</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @project_aliases.each do |project_alias| %>
+      <tr>
+        <td><%= project_alias.name %></td>
+        <td><%= project_alias.target_type %></td>
+        <td><%= link_to(project_alias.target.try(:name), project_alias.target) %></td>
+        <td>
+          <%= link_to(:EDIT.t, edit_project_alias_path(project_id:, id: project_alias.id)) %>
+          <%= link_to(:DELETE.t, project_alias_path(project_id:, id: project_alias.id),
+	              data: { turbo_method: :delete, turbo_confirm: :are_you_sure.t }) %>
+        </td>
+      </tr>
```

Comment:
Maybe worth checking out our `TableHelper`, that predates me.

You can iterate over your `@project aliases` in the controller or the template, generating a simple array of content. Then call `make_table` with this array and it renders all that HTML.

The reason I've been haphazardly trying to convert layouts to use it, is because it will be helpful to have the HTML centralized for the bootstrap upgrade. Applying styling can happen in the helper, rather than tracking down every table in the layouts.

Reply:
Done.